src/stateless/ScrollableList.vue

```diff
@@ -56,6 +57,22 @@ export default {
         showOverlay: { type: Boolean, default: false },
         initialOffset: { type: Number, default: 0 },
     },
+    setup () {
+        // Wait for large items to render
+        return new Promise(resolve => setTimeout(resolve, 200))
```

Comment:
This stuff with settimeout is not really ideal, can we find a better way? Maybe hook to a lifecycle event or sth?